src/Core/tests/DeviceTests.Shared/HandlerTests/HandlerTestBasement.cs

```diff
@@ -127,6 +131,27 @@ protected void InitializeViewHandler(IElement element, IElementHandler handler,
 #endif
 
 				view.Arrange(new Rect(0, 0, w, h));
+
+#if WINDOWS
+				if (viewHandler is SwipeViewHandler)
```

Comment:
Instead of checking the handler type should we check the platformview type?

if (viewHandler.PlatformView is SwipeControl) ?

Reply:
Yeah, that check will work too. But, is not more generic the handler one?. I mean, if the PlatformView or something change in the handler impl using the handler to check will continue working.

Reply:
The exception is particular to the SwipeControl yea? So, if the SwipeViewHandler changes to a different PlatformView then I think this code is irrelevant.

Reply:
Changed.